### PR TITLE
Add Desktop Media Queries

### PR DESCRIPTION
**Pull request by Lornakaboro**
This is a portfolio that is aimed at showcasing my accomplishments  ,skills , educational, work backgrounds and contact form.
I added a media query for desktop size screen

### Getting Started

To get a local copy up and running follow these simple example steps.

### Prerequisites

- Git installed in desktop
- Code editor of your choice i.e; Visual Studio Code
- Browser of your choice i.e; Mozilla Firefox
- Terminal of your choice i.e; Git Bash

### Setup

To get a local copy up and running follow these simple example steps.
- use the git clone command with this [link](git@github.com:Lornakaboro/my-portfolio.git)
- cd into my-portfolio
- Switch branch using this command `git checkout create-portfolio-desktop-version`
- Open index.html in your browser
- You will be able to see a template with a contact form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,7 +12,12 @@
...
     <header class="header">
       <a href="" class="logo">My Logo</a>
-      <i class="fa fa-bars" aria-hidden="true"></i>
+      <i class="fa fa-bars hamburger" aria-hidden="true"></i>
+      <ul class="nav">
+        <li class="nav-item"><a href="!#" class="nav-link">Portfolio</a></li>
+        <li class="nav-item"><a href="!#" class="nav-link">About</a></li>
+        <li class="nav-item"><a href="!#" class="nav-link">Contact</a></li>
+      </ul>
     </header>
```
Comment (Bondok6) [inline index.html:21]: - [ ] Great job implementing the Desktop version and Everything is perfect 💯 However, You should make sure the mobile design still looks as expected, so kindly make sure that navigation items do not appear in the mobile version👌




- [ ] Also, make sure that the style of the cards in the `my-works` section works as expected on the mobile version🙏

File: style.css [hunk 2]
```diff
@@ -340,4 +330,130 @@ body {
...
+    font-weight: 500;
+  }
+
+  .contact-form {
+    background-image: url("form-background.png");
+    background-size: cover;
+    height: 1000px;
+    background-repeat: no-repeat;
+    background-color: #fff;
+  }
```
Comment (Bondok6) [inline style.css:449]: - [ ] According to the requirements, You must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). So, Kindly make sure the background-image is similar to your Figma design and decrease the margin-top between the heading `<h2>` and the paragraph `<p>`💯

| Your Design  | Figma Design  |   
| --- | --- |
|   |  |

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @Lornakaboro 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights
- No linter errors ✅
- Your repo is professional ✅
- Using Gitflow correctly ✅

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Lornakaboro) [reply index.html:21]: @Bondok6 , I have fixed mobile view of my-works section styling in media query of 480px and below.

I have also enabled disappearing of the navigation items in the mobile view.

File: style.css [hunk 3]
```diff
@@ -308,13 +295,30 @@ body {
...
     background-image: url("background.png");
     background-size: cover;
     background-repeat: no-repeat;
     height: 779px;
   }
 
+  .work {
+    margin: 10% 5% 10% 5%;
+  }
+
```
Comment (franclobo) [inline style.css:321]: - [ ] **[OPTIONAL]** Please 🙏🏽 the mobile version should continue working as expected.
image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Lornakaboro, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Professional README file ✔️ 
- Correct Github flow ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] Please 🙏🏽 the desktop and the mobile version should work as expected, for a better user eperience.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @Lornakaboro  ,  :wave: 
### I'm Levy, your PR reviewer

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

[OPTIONAL] Kindly adjust your stack list element to look exactly like the Figma design. :wink: 
      
  Your design
            
  image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**